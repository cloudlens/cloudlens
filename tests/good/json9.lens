group upon (e.status){
    "^Start"
}

stream {
    print("----")
}

stream (e in log.group){
    print("verses: " + e.message);
}
