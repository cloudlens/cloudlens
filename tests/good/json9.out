----
verses: eeny
verses: meeny
verses: miny
verses: moe
----
verses: ams
verses: tram
verses: gram
----
verses: pinto
verses: pinto
verses: gorgorito
